pko/package/manifest.yaml

```diff
@@ -43,11 +44,11 @@ spec:
             imageTag:
               type: string
               description: tag of the image containing unleash
-              default: UNLEASH_IMAGE_TAG
+              default: latest
```

Comment:
UNLEASH_IMAGE_TAG was meant to be a default to keep the PKO package and the unleash release in sync. it is a variable replaced by the 7digit sha during the build of the package.

what is the reason to replace it with latest?

Reply:
Was for local testing only